src/config.xml

```diff
@@ -1815,16 +1815,19 @@ to disable this feature.
     <option type='int' id='MAX_ITEMS_BEFORE_QUICK_INDEX' minval='0' maxval='3000' defval='30'>
       <docs>
 <![CDATA[
- MAX_ITEMS_BEFORE_QUICK_INDEX defines, when to set the global lists of functions, 
- variables, typdefs, enums, macros grouped into sections by starting letter. Default value: 30
+ If the number of items on al list of functions, variables, typdefs, enums and macros 
+ exceeds \c MAX_ITEMS_BEFORE_QUICK_INDEX, it will be divided into multiple sections
+ to form a quick index. If the number of items exceeds \c MAX_ITEMS_BEFORE_MULTIPAGE_INDEX, 
+ those sections will be put on a single output page each.
```

Comment:
Better, though:
- first line `al` should be `a`
- first line `and` shouldn't this be `or` or should there be something like the word `combined` in the sentence.
- `\c MAX_ITEMS_BEFORE_MULTIPAGE_INDEX` should be `\ref cfg_max_items_before_multipage_inxdex "MAX_ITEMS_BEFORE_MULTIPAGE_INDEX"`
- last sentence, I think the word `each` shouldn't be present.